doc/usage.adoc

```diff
@@ -59,6 +59,11 @@ python rules2yml.py # Generates the rule directory
 osivalidator --data data/20240221T141700Z_sv_300_2112_10_one_moving_object.osi --rules rules -p
 ----
 
+The rules2yml.py generates yml files for all OSI proto files containing the rules specified in OSI.
+It also generated respective yml schema files to validate the rule yml files.
+The schema files are located in <rules_folder>/schema.
+When executing osivalidator, all rule files are validated against their respective schema.
```

Comment:
It parses the rules in combination with their application in the original osi files. Why would the schema be coupled to the filename? Isn't the base idea that someone creates custom yml files (setting a speedlimit on an object) and edits existing ones and wants a checker for those?

Reply:
The schema files do not contain rules for specific fields. All rules are allowed for all fields. So having custom rules files is supported.

Reply:
I adjusted the documentation.